mobile/src/main.jsx

```diff
@@ -66,6 +66,7 @@ document.addEventListener('DOMContentLoaded', () => {
         })
       } else {
         initClient(url)
+        cozy.client.offline.startRepeatedReplication('io.cozy.files', 15)
```

Comment:
If we set up the `replication` workflow in a way (cf. https://github.com/cozy/cozy-files-v3/pull/142/files#r104361067), don't we have to start it here the same way: first `replicateFromCozy` then `startRepeatedReplication`?